src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/integration/TestContainersIntegrationTestBase.kt

```diff
@@ -0,0 +1,79 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.integration
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.awaitility.Awaitility
+import org.awaitility.kotlin.await
+import org.awaitility.kotlin.matches
+import org.awaitility.kotlin.untilCallTo
+import org.junit.jupiter.api.BeforeEach
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.DynamicPropertyRegistry
+import org.springframework.test.context.DynamicPropertySource
+import software.amazon.awssdk.services.sqs.model.PurgeQueueRequest
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.integration.container.LocalStackContainer
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.integration.container.LocalStackContainer.setLocalStackProperties
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.events.DomainEvent
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.events.EventType
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.events.Message
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.events.MessageAttributes
+import uk.gov.justice.hmpps.sqs.HmppsQueue
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+import uk.gov.justice.hmpps.sqs.countAllMessagesOnQueue
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+@ActiveProfiles("test-localstack", inheritProfiles = false)
+abstract class TestContainersIntegrationTestBase : IntegrationTestBase() {
```

Comment:
This might be better renamed to something else?  SQS/SNS/Message based.

Reply:
Yeah, TestContainersIntegrationTestBase does not really give a clue that its for localstack-based tests.

Reply:
Done